rr_iarrc/src/cone_detection/cone_detection_height.cpp

```diff
@@ -0,0 +1,336 @@
+#include <ros/ros.h>
+#include <ros/publisher.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/Image.h>
+#include "std_msgs/Float64MultiArray.h"
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/opencv.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/opencv.hpp>
+#include <sensor_msgs/PointCloud2.h>
+#include <pcl_conversions/pcl_conversions.h>
+#include <boost/asio.hpp>
+#include <boost/algorithm/string.hpp>
+#include <boost/math/constants/constants.hpp>
+#include <algorithm>
+#include <pcl_ros/point_cloud.h>
+#include <iostream>
+#include <stdlib.h>
+#include <stdio.h>
+#include <sensor_msgs/CameraInfo.h>
+
+using namespace cv;
```

Comment:
We would prefer using minimal namespaces. Std is fine, cv and ros should probably not be used.